C++/Logger/TypeIs.h

```diff
@@ -2,19 +2,19 @@
 #define _TYPE_IS_H_
 
 template <typename T>
-struct type_is
+struct TypeIs
```

Comment:
I too prefer `type_is`; I was conforming to the style of the existing code base, shall I switch it back?